test/XchngToken.test.js

```diff
@@ -264,6 +276,94 @@ contract('XchngToken', async ([ownerAddress,recipient,anotherAccount]) => {
     });
     // Test tranfers from one account to another 
     describe('transferFrom()', function() {
+        const spender = recipient;
+        beforeEach(async function () {
+            // Give the approver a balance of 100
+            await this.token.transfer(approver, 100, { from: ownerAddress });
```

Comment:
Why do this before each here with the ownerAddress and in the nested describes ? Seems like you could just set this to approver address and use it for all tests below.